components/Animations/src/Xaml/AnimationSet.cs

```diff
@@ -57,7 +57,7 @@ public sealed class AnimationSet : DependencyObjectCollection
     /// <summary>
     /// Gets or sets the weak reference to the parent that owns the current animation collection.
     /// </summary>
-    internal WeakReference<UIElement>? ParentReference { get; set; }
+    public WeakReference<UIElement>? ParentReference { get; set; }
```

Comment:
These seems like a case where `protected` is probably the better intent over public as this is an internal holder of a reference and not something anyone should just get at/use.

Wish we could easily see references from PRs on GitHub and what's using this... 😋

Reply:
> Wish we could easily see references from PRs on GitHub and what's using this... 😋

You can do this from the general file browser on github, just select the right branch and file. Works from the "Files Changed" tabs in PRs, too.